lib/ui/window.dart

```diff
@@ -166,10 +201,19 @@ class Locale {
   /// Locale('he')` and `const Locale('iw')` are equal, and both have the
   /// [languageCode] `he`, because `iw` is a deprecated language subtag that was
   /// replaced by the subtag `he`.
-  String get languageCode => _canonicalizeLanguageCode(_languageCode);
+  ///
+  /// This should be a valid Unicode Language subtag as listed in [Unicode CLDR
+  /// supplemental
+  /// data](http://unicode.org/cldr/latest/common/validity/language.xml).
```

Comment:
maybe mention that it won't be null, but might be `und`

Reply:
I think this is implied by "should be a valid Unicode Language subtag"?